daml-assistant/daml-helper/src/DA/Daml/Helper/Main.hs

```diff
@@ -94,25 +94,33 @@ commandParser = subparser $ fold
         <*> optional (option str (long "on-start" <> metavar "COMMAND" <> help "Command to run once sandbox and navigator are running."))
         <*> (WaitForSignal <$> flagYesNoAuto "wait-for-signal" True "Wait for Ctrl+C or interrupt after starting servers." idm)
 
-    deployCmdInfo = progDesc . concat $
-        [ "Deploy the current DAML project to a remote DAML ledger. "
-        , "This will allocate the project's parties on the ledger "
-        , "(if missing) and upload the project's built DAR file. You "
-        , "can specify the ledger in daml.yaml with the ledger.host and "
-        , "ledger.port options, or you can pass the --host and --port "
-        , "flags to this command instead."
+    deployCmdInfo = mconcat
+        [ progDesc $ concat
```

Comment:
Just an idea for the future: using `unwords` instead of `concat` allows for skipping the trailing spaces.

Reply:
Ah yes, I had done this already in my PR for this same issue, but Moritz's PR was slightly better (and we submitted them at the same time) :-)